rumqttc/src/tls.rs

```diff
@@ -51,23 +49,26 @@ pub enum Error {
     #[error("TLS error: {0}")]
     TLS(#[from] rustls::Error),
     #[cfg(feature = "use-rustls")]
-    /// No valid certificate in chain
-    #[error("No valid certificate in chain")]
+    /// No valid CA cert found
+    #[error("No valid CA certificate provided")]
     NoValidCertInChain,
+    #[cfg(feature = "use-rustls")]
+    /// No valid client cert found
+    #[error("No valid certificate for client authentication in chain")]
+    NoValidClientCertInChain,
+    #[cfg(feature = "use-rustls")]
+    /// No valid key found
+    #[error("No valid key in chain")]
+    NoValidKeyInChain,
     #[cfg(feature = "use-native-tls")]
     #[error("Native TLS error {0}")]
     NativeTls(#[from] NativeTlsError),
 }
 
-// // The cert handling functions return unit right now, this is a shortcut
-// impl From<()> for Error {
-//     fn from(_: ()) -> Self {
-//         Error::NoValidCertInChain
-//     }
-// }
-
 #[cfg(feature = "use-rustls")]
 pub async fn rustls_connector(tls_config: &TlsConfiguration) -> Result<RustlsConnector, Error> {
+    use rustls_pemfile::Item;
```

Comment:
can we please move this import to top ( where other imports are ) to be consistent with imports? Don't forget to feature gate it though haha

PS: I should have added this comment in last review but I forgot, my bad :sweat_smile:

Reply:
Ah yes, auto import placed it there. good catch